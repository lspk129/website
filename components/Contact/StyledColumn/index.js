import styled from 'styled-components';
import { Column } from 'hedron';

const StyledColumn = styled(Column)`
  background: ${props => props.theme.secondaryBackground};
`;

export default StyledColumn;
