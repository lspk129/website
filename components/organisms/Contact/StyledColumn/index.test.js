import React from 'react';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';
import { Column } from 'hedron';

import StyledColumn from './';

describe('StyledColumn', () => {
  const shallowWrapper = shallow(<StyledColumn />);

  it('renders correctly', () => {
    const wrapper = renderer.create(<StyledColumn />);
    const tree = wrapper.toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('renders Column component', () => {
    expect(shallowWrapper).toBePresent();
    expect(shallowWrapper.find(Column)).toBePresent();
  });
});
