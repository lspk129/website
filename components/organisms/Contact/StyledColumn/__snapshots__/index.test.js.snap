// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StyledColumn renders correctly 1`] = `
<div
  className="sc-ifAKCX kYQwmh sc-htpNat bKOCBb"
/>
`;
